README.md

```diff
@@ -54,3 +106,149 @@ Small note: If editing the README, please conform to the [standard-readme](https
 ## License
 
 This repository is only for documents. All of these are licensed under the [CC-BY-SA 3.0](https://ipfs.io/ipfs/QmVreNvKsQmQZ83T86cWSjPu2vR3yZHGPm5jnxFuunEB9u) license, © 2016 Protocol Labs Inc.
```

Comment:
License should just be `CC-BY`.

Reply:
This was unchanged from the prior version. Did the new IP Policy land yet @ianjdarrow?

Reply:
I'm going to resolve this for now as I still haven't heard from @ianjdarrow and I'm not comfortable changing the license until I do.